.github/workflows/ci.yml

```diff
@@ -24,6 +24,9 @@ jobs:
       - name: Set up Rprofile
         run: cp Docker.Rprofile $HOME/.Rprofile
         shell: bash
+      - name: Install SEIR package
+        run: /home/app/python_venv/bin/python setup.py install
+        shell: bash
```

Comment:
Not necessary for this PR, but can this be combined with the local_install.R?

Reply:
what do you mean combined? i can have them run in 1 CI step with `&&` but would rather work on a solution that obviates the need for `local_install.R`

Reply:
I was thinking of combining into one script, but getting of local_install.R later on would be much, much better.

Reply:
local_install.R is not necessary for the docker now, so i think removing it from ci should be straightforward